docs/services/gracc-corrections.md

```diff
@@ -0,0 +1,62 @@
+Installing GRACC Corrections
+============================
+
+GRACC Corrections are used to modify records during the summarization process.  RAW records are not modified in the correction process.  The correction is applied after summarization and aggregation, but before the record is enriched with data from [Topology](https://topology.opensciencegrid.org/).
+
+The correction is step 3 in the GRACC summary record workflow:
+
+1. Raw record is received.  The raw record is never modified
+2. Summarizer aggregates the raw records
+3. Corrections are applied
+4. Summarized records are enriched by Topology
+5. Summarized and enriched records are uploaded to GRACC
+
+We can currently correct:
+
+* VO Names
+* Project Names
+* OIM_Site (using the `Host_description` field)
+
+Limitations
+-----------
+
+Additional corrections can be written, but some attributes are protected.  GRACC uses specific attributes to detect duplicate records.  Protected records for summarization are:
+
+    EndTime, RawVOName, RawProjectName, DN, Processors, ResourceType, CommonName,
+    Host_description, Resource_ExitCode, Grid, ReportableVOName, ProbeName
+
+For example, we could not write a correction for the `Host_description`.  If we had a correction that changed `Host_description`, then the duplicate detection would not detect the same record during resummarization and it would have duplicates summarized records.
+
+Command Line
+------------
+
+The [gracc-correct](https://github.com/opensciencegrid/gracc-tools/tree/master/gracc-correct) tool is used to create, update, and delete corrections.  The tool must be run from a host that can write to GRACC, which is very restricted.  It is recommended to run the _gracc-correct_ tool directly from the gracc.opensciencegrid.org host.
+
+The _gracc-correct_ tool is able to parse new corrections from either user input or many from a CSV file.
```

Comment:
> from either user input or many from a CSV file

something feels awkward about this phrasing ...

maybe you mean something like, "either [individually] from user input or many [at once] from a CSV file"

(More generally: "from either X or Y" or "either from X or from Y" ... but "from either X or from Y" sounds just a bit awkward)